src/c-writer.cc

```diff
@@ -1967,7 +2023,8 @@ void CWriter::WriteParamTypes(const FuncDeclaration& decl) {
 
 void CWriter::WriteLocals(const std::vector<std::string>& index_to_name) {
   Index num_params = func_->GetNumParams();
-  for (Type type : {Type::I32, Type::I64, Type::F32, Type::F64}) {
+  for (Type type : {Type::I32, Type::I64, Type::F32, Type::F64, Type::FuncRef,
+                    Type::ExternRef}) {
```

Comment:
Is the some way to C++ to say `for (Type type : AllTypeEnumValues)` to avoid spelling this out here and below?

Reply:
Unfortunately I don't know of a _great_ way, and it's made more complicated by the fact that we're skipping a bunch of Type enum values that aren't meaningful or supported here (e.g. I8, Func, Void, Reference, Any, V128...).